src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -232,10 +150,9 @@ internal void Validate()
             // Each storage provider may have its own limitations for task hub names due to provider naming restrictions
             this.StorageProvider.GetConfiguredProvider().ValidateHubName(this.HubName);
 
-            if (this.EventGridPublishRetryInterval <= TimeSpan.Zero ||
```

Comment:
Can we add a call to `this.Tracing.Validate()'`?

Reply:
Ah, I didn't realize `this.Tracing.Validate()` doesn't exist. I'm debating whether it would be good to add that method right now to reenforce the pattern and make it easier to add validation later if we need it, but you can probably leave this as is for now.